### PR TITLE
Folders

### LINKED ISSUE #832: Set folder for RemoteDataset
## Detailed Description
- Affected library: webknossos

webKnossos now supports dataset management in the form of folders. It would be great to be able to assign folders using the `RemoteDataset` class.

## Use Cases & Context
- Import scripts e.g. with PublishDataset in Voxelytics

### PR DESCRIPTION
**Pull request by normanrz**
### Description:
- Adds support to assign folders to RemoteDatasets

### Issues:
- fixes #832 

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [x] Updated Changelog
 - [x] Updated Documentation
 - [x] Added / Updated Tests

### CODE REVIEW

**Review by jstriebel — APPROVED** [review]
:tada:

**Comment by fm3** [comment]
I updated the snapshots. I don’t know why the CI format check fails, the format script yields no changes locally. I executed make install before ./format.sh locally. Does that work for you?

**Comment by jstriebel** [comment]
@normanrz Tests are green now. Please add a changelog entry, and maybe also more docs or examples if you want to.

File: webknossos/webknossos/dataset/dataset.py [hunk 1]
```diff
@@ -1958,3 +1963,12 @@ def allowed_teams(self, allowed_teams: Sequence[Union[str, "Team"]]) -> None:
...
                 dataset_update_teams_response.status_code == 200
             ), dataset_update_teams_response
+
+    @property
+    def folder(self) -> RemoteFolder:
+        return RemoteFolder.get_by_id(self._get_dataset_info().folder_id)
+
+    @folder.setter
+    def folder(self, folder: RemoteFolder) -> None:
+        """Move the dataset to a folder. Specify the folder like this `RemoteFolder.get_by_path("Root/Folder_A")`."""
```
Comment (jstriebel) [inline webknossos/webknossos/dataset/dataset.py:1973]: I think the root is called `Datasets` by default?

```suggestion
        """Move the dataset to a folder. Specify the folder like this `RemoteFolder.get_by_path("Datasets/Folder_A")`."""
```

**Review by jstriebel — APPROVED** [review]
:+1:

File: webknossos/local_wk_setup.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 function export_vars {
     export WK_TOKEN=1b88db86331a38c21a0b235794b9e459856490d70408bcffb767f64ade0f83d2bdb4c4e181b9a9a30cdece7cb7c65208cc43b6c1bb5987f5ece00d348b1a905502a266f8fc64f0371cd6559393d72e031d0c2d0cabad58cccf957bb258bc86f05b5dc3d4fff3d5e3d9c0389a6027d861a21e78e3222fb6c5b7944520ef21761e
     export WK_URL=http://localhost:9000
-    export DOCKER_TAG=master__20617
+    export DOCKER_TAG=master__10303
```
Comment (jstriebel) [inline webknossos/local_wk_setup.sh:4]: @normanrz Just noticed that this is not a valid tag, I'm updating it now in another PR to [master__21723](https://hub.docker.com/layers/scalableminds/webknossos/master__21723/images/sha256-22d498777f8061fa653a9590c4a56f50a1c5e80ac4b433b60495f351c68534a8?context=explore)

**Pull request merged**